build-system/tasks/amp-task-runner.js

```diff
@@ -0,0 +1,152 @@
+/**
+ * Copyright 2021 The AMP HTML Authors. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS-IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/**
+ * @fileoverview This is a lightweight task runner with a one-file implentation
+ * based on the commander npm package. It behaves like gulp without the overhead
```

Comment:
I don't think it's necessary mention `gulp` in the JSDoc - in a year it'll look apocryphal :)

Reply:
Heh! Agree. Removed.